### PR TITLE
PLAT-10369: User Service implementation.

### PR DESCRIPTION
**Pull request by symphony-hong**
### Ticket
[PLAT-10369](https://perzoinc.atlassian.net/browse/PLAT-10369)

### Description
Create UserService class wrapping all the service api related to message service which are documented in the developers.symphony.com site.

Create method in the SymphonyBdk class to initialize the UserService instance and provide it to bot developer to work with message service.

Create an example which shows how a bot interact with the user service.

Unittest added.

Documentation added.

### Dependencies
List the other pull requests that should be merged before/along this one.

### Checklist
- [x] Referenced a ticket in the PR title and in the corresponding section
- [x] Filled properly the description and dependencies, if any
- [x] Unit tests updated or added
- [x] Docstrings added or updated
- [x] Updated the documentation in [docs folder](../docs)

### CODE REVIEW

**Review by symphony-youri — COMMENTED** [review]
For the Lenna picture let's maybe use another image (https://en.wikipedia.org/wiki/Lenna#Criticism) (and we would have to change it in the Java BDK too)

File: symphony/bdk/core/service/user/user_service.py [hunk 1]
```diff
@@ -0,0 +1,704 @@
...
+                            returned. If absent or false then a directory search will be performed and users
+                            from other pods who are visible to the calling user will also be returned.
+        :param active:      If not set all user status will be returned,
+                            if true all active users will be returned,
+                            if false all inactive users will be returned.
+
+        :return: Users found by user ids.
+
+        """
+        user_ids_str = ','.join([str(user_id) for user_id in user_ids])
```
Comment (symphony-youri) [inline symphony/bdk/core/service/user/user_service.py]: I find this version: https://stackoverflow.com/a/3590175 (using map) even shorter and easier to read

**Comment by symphony-elias** [comment]
There are several endpoints missing:
- https://developers.symphony.com/restapi/v20.10/reference#list-disclaimers
- https://developers.symphony.com/restapi/v20.10/reference#disclaimer-users
- https://developers.symphony.com/restapi/v20.10/reference#user-disclaimer
- https://developers.symphony.com/restapi/v20.10/reference#suspend-user-v1
Please check if it is feasible to add these endpoints. We may have to regenerate the the python modules in bdk.gen following changes in https://github.com/symphonyoss/symphony-api-spec

**Review by symphony-elias — APPROVED** [review]
LGTM

File: examples/core/service/user_main.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+from symphony.bdk.core.config.bdk_config_loader import BdkConfigLoader
+from symphony.bdk.core.symphony_bdk import SymphonyBdk
+
+
+class UserMain:
+
+    @staticmethod
+    async def run():
+
+        config_3 = BdkConfigLoader.load_from_symphony_dir("config.yaml")
```
Comment (thibauult) [inline examples/core/service/user_main.py]: Why not just `config`?

**Pull request merged**